scan.py

```diff
@@ -10,6 +10,10 @@
 # Returns all the text in a file as a list
 # Return an empty list if no text is found
 def get_file_text(file) -> list:
+    """
```

Comment:
It's OK if you want to do this, however, please make sure that rtype matches the type hint (-> list)

Reply:
thank you !